docker/version_keys.py

```diff
@@ -0,0 +1,5 @@
+version_keys = {
```

Comment:
1. maybe rename to version_gpg_keys
2. missing license
3. Can we add comment to the file about the format of this file.

Reply:
Thanks for the review @omkreddy .
1. Renamed this to version_gpg_keys, the name now better reflects what the file stands for.
2, 3: Sorry, missed out on adding these! Added the same to the scripts.
Thanks!